src/Draco.Compiler/Internal/Semantics/SymbolResolutionErrors.cs

```diff
@@ -0,0 +1,30 @@
+using Draco.Compiler.Api.Diagnostics;
+
+namespace Draco.Compiler.Internal.Semantics;
+
+/// <summary>
+/// Holds constants for symbol resolution errors.
+/// </summary>
+internal static class SymbolResolutionErrors
+{
+    private static string SymbolResolutionErrorCode(int index) => DiagnosticTemplate.CreateErrorCode(ErrorCategories.SymbolResolutionError, index);
```

Comment:
`ErrorCode`, you used that name in codegen, and I'd agree it's redundant here to call it `SymbolResolutionErrorCode`